c2/c2-protocol/c2-protocol-component-api/pom.xml

```diff
@@ -15,9 +15,9 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
  -->
-<project xmlns="https://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="https://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="https://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
These look like an IDE did that and IIRC we were supposed to have them use HTTPS?

Reply:
My IntelliJ couldn't resolve this one as https and others looked liked http so I switched, but if we think that's incorrect then I can switch it back.